packages/sdk/src/evm/core/classes/erc-1155-lazymintable.ts

```diff
@@ -232,17 +230,10 @@ export class Erc1155LazyMintable implements DetectableFeature {
   }
 
   private async isLegacyEditionDropContract() {
-    if (hasFunction<TokenERC721>("contractType", this.contractWrapper)) {
-      try {
-        const contractType = ethers.utils.toUtf8String(
-          await this.contractWrapper.readContract.contractType(),
-        );
-        return contractType.includes("DropERC1155");
-      } catch (e) {
-        return false;
-      }
-    } else {
-      return false;
-    }
+    const info = await getPrebuiltInfo(
+      this.contractWrapper.readContract.address,
+      this.contractWrapper.getProvider(),
+    );
+    return info && info.type === "DropERC1155" && info.version <= 2;
```

Comment:
Should this be `<3` instead of `<=2` for version - do we do incremental version changes like `2.0.1` on contracts (assuming not, but figured I would mentioned)

Reply:
yeah, we don't do decimals for versions, but good point